driver-core/src/main/com/mongodb/internal/connection/BaseCluster.java

```diff
@@ -343,14 +350,13 @@ private static List<ServerTuple> atMostNRandom(final ArrayList<ServerDescription
         return result;
     }
 
-    private ServerSelector getCompositeServerSelector(final ServerSelector serverSelector) {
+    private ServerSelector getCompleteServerSelector(final ServerSelector serverSelector, final ServerDeprioritization serverDeprioritization) {
```

Comment:
Removing the subclass of the return type from the method name is fine, but it seems "Complete" has no implication except "Actual" (there is no incomplete selector in this class?). Let's use `getServerSelector`.

Reply:
> there is no incomplete selector in this class?

There is - the `serverSelector` passed to the `getCompleteServerSelector` method.